### PR TITLE
Init from etcd: basic support

### PR DESCRIPTION
**Pull request by reshke**
change router initialization in clustered setup mechanics.
Router requests current metadata setup from qdb on start. Router sync metadata routine now sync only current coordinator address for router proxy queries.

### CODE REVIEW

**Review by Denchick — CHANGES_REQUESTED** [review]
It would be nice to add tests that check whether or not the initialization of metadata for different types of installations actually works.

File: examples/coordinator-init.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+log_level: debug
```
Comment (Denchick) [inline examples/coordinator-init.yaml:1]: it looks like a router config, is it definitely coordinator-init.yaml?

**Review by Denchick — APPROVED** [review]

**Review by Denchick — CHANGES_REQUESTED** [review]
Add comments

File: cmd/router/main.go [hunk 2]
```diff
@@ -317,6 +317,26 @@ var runCmd = &cobra.Command{
 			}
 		}()
 
+		/* initialize metadata */
```
Comment (Denchick) [inline cmd/router/main.go:320]:
```suggestion
		// The router will execute the file "init_sql" before starting.
```

**Review by Denchick — APPROVED** [review]

File: examples/coordinator-init.yaml (hunk 1 above)
Comment (reshke) [reply examples/coordinator-init.yaml:1]: it is router-etcd-init.yaml

File: cmd/router/main.go (hunk 2 above)
Comment (reshke) [reply cmd/router/main.go:320]: no, this is both init sql and coodinator init cases here

**Pull request merged**